packages/beacon-node/src/chain/seenCache/seenAttestationData.ts

```diff
@@ -11,8 +11,8 @@ type AttDataBase64 = string;
 export type AttestationDataCacheEntry = {
   // part of shuffling data, so this does not take memory
   committeeValidatorIndices: Uint32Array;
-  // TODO: remove this? this is available in SingleAttestation
   committeeIndex: CommitteeIndex;
+  attesterIndex: ValidatorIndex;
```

Comment:
it's not correct to have `attesterIndex` here, this is the common cache for the whole committee while this is specific information of a validator

should also remove `aggregationBits` below as it's specific to the validator